antora-playbook.yml

```diff
@@ -8,6 +8,11 @@ site:
 # see https://docs.antora.org/antora/2.3/playbook/configure-runtime/
 runtime:
   cache_dir: ./.cache/antora
+  log:
+    # use pretty even on CI
+    format: pretty
+    # set to info to get details from the Antora extensions
+    level: info
```

Comment:
Did you try the `failure-level` key? 
I thing we should use it, with error (or warn?) to ensure that the content of the pages is correctly converted into HTML by Antora. 
Today if there is a syntax error it fails silently, i would like to change this.

https://docs.antora.org/antora/latest/playbook/configure-runtime/#log-key

Reply:
Yes I tested it.
We cannot set `warn` for now as there are a lot of content issues in all repositories (especially broken xref and buggy attribute substitutions) that would need to be fixed first and this is out of the scope here. But I agree that this should be our target.

We could however set it to `error` but I am not sure whether if we have some now or not. I will try it with the whole site but we could have issue when building preview for documentation content PR.
Anyway, we could set the error level and rollback if it blocks the PR.

Reply:
see 6f4c794

Reply:
Tested and setting it to error level make the build fail with the preview of this PR, there are a lot of broken links 👿 
```
[14:49:14.312] ERROR (asciidoctor): target of xref not found: 2021.2@bonita::pause-and-resume-bpm-services.adoc
    file: modules/ROOT/pages/livingapp_deploy.adoc:170
    source: bonitasoft/bonita-continuous-delivery-doc.git (refname: 3.5)
[14:49:14.328] ERROR (asciidoctor): target of xref not found: 2021.2@bonita::environments.adoc
    file: modules/ROOT/pages/livingapp_manage_configuration.adoc:12
    source: bonitasoft/bonita-continuous-delivery-doc.git (refname: 3.5)
```